snapcraft/internal/elf.py

```diff
@@ -531,10 +532,10 @@ def _get_system_libs() -> FrozenSet[str]:
         logger.debug('Only excluding libc libraries from the release')
         libc6_libs = [os.path.basename(l)
                       for l in repo.Repo.get_package_libraries('libc6')]
-        return frozenset(libc6_libs)
-
-    with open(lib_path) as fn:
-        _libraries = frozenset(fn.read().split())
+        _libraries = frozenset(libc6_libs)
+    else:
+        with open(lib_path) as fn:
+            _libraries = frozenset(fn.read().split())
```

Comment:
I fail to see how this reduces the number of calls. The change seems equivalent to me, so I might be missing something.
I mean, +1 because it looks correct, but can you explain it to me in case I'm doing a wrong review, missing something obvious because I'm being dense?

Reply:
it is at the beginning of the function:
```python
global _libraries
     if _libraries:  # type: ignore
return _libraries # type: ignore
```

Reply:
Ahh, I get it now. Thanks.